intro-to-git/collaborating.md

```diff
@@ -0,0 +1,171 @@
+
+- The project should be hosted somewhere, where team members have access. The project will can be called the project "repo."
+- Each developer will download the repo onto their machine using the Git command `clone`. This "download" will carry the Git history
+- All team members should contribute code changes by making commits on their own machines
+- All team members will receive commits and updates from the main hosted repo
+- All team members will send commits and updates to the main hosted repo
+- All team members will follow and repeat this cycle, enabling a smooth team workflow
+
+### Example: The Hike Planner Team
+
+For the Hike Planner team, we might imagine:
+
+- Nakita will host the project codebase somewhere, and give access to Frida and Raul
+- Frida will clone this repo onto her laptop. Raul will clone this repo onto his laptop.
+- Nakita will make code changes and commits on her machine. Frida will make other code changes and commits on her machine. Raul will, too.
+- When Nakita wants the updates and new commits that are on the main hosted repo, she will ask for them. These updates and commits will be applied to the code she currently has on her laptop.
+    - When Frida wants updates, she'll ask for them too. The updates/commits will be applied to her current code.
+    - When Raul wants updates, he'll ask for them, and the updates will be applied to his current code.
+- When Nakita wants to send commits that are on her laptop to the main hosted repo, she will push her commits. This process will use Git's merging tools, and the main hosted repo will now have a new, updated Git history.
+    - When Frida pushes her commits, the main hosted repo will also work to have a new, updated Git history.
+    - Raul's commits and contributions work the same way, too.
+- All three team members will contribute through making commits, pulling commits from the main repo, and pushing commits to the main repo.
+
+### The Remote Repository and `origin`
+
+When we mention the "main hosted repo," we need to find a host. In most cases, this decision is already made and the host of this code base is Github.com.
+
+A repo that holds a codebase and commit history that is accessible to team members is called a **remote repository.**
+
+This repo is usually hosted somewhere that is _not_ a developer's personal computer.
+
+What do we do with a remote repo? All team members will look to this one, shared remote repo. It will be the main repo that team members can use as a source of truth. Team members will get new commits and updates from the remote repo. Team members will push their new commits to the remote repo.
+
+#### Default Settings
+
+Often, the remote repo will be hosted on Github.com. Git usually gives a nickname to the remote repo hosted on Github.com: `origin`. There are many Git commands that will need a reference to a remote repo; we can use `origin` to mean the remote repo on Github.com
```

Comment:
Add a line that can help explain the name `origin`. Perhaps, "When people first clone the repo, they will do some from the remote on Github.com. Hence why we call it `origin`, since that's where people will originally get the code!"